### PR TITLE
Improve Bug Template

### PR DESCRIPTION
**Pull request by Sweetdevil144**
# Resolves HASH_SIGN_FOLLOWED_BY_ISSUE_NUMBER

## What is being addressed

Describe the current behavior you are modifying. Please also remember to update any impacted documentation.

I noticed the bug template has numeration mistakes. So I'm now correcting them with this PR.

## How is this addressed

- Update documentation

### CODE REVIEW

**Comment by tim-allen-ck** [comment]
@Sweetdevil144 Could you update the changelog?

**Comment by Sweetdevil144** [comment]
Change Reports added to `Changelog`.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ BUG FIXES:
 * Fixes broken links on 'Using the Azure TRE -> Custom Templates' page of documentation ([[#4003](https://github.com/microsoft/AzureTRE/issues/4003)])
 * Fix 'Renew Lets Encrypt Certificates' GitHub Action ([#3978](https://github.com/microsoft/AzureTRE/issues/3978))
 * Add lifecycle rule to the Gitea Shared Service template for the MySQL resource to stop it recreating on `update` ([#4006](https://github.com/microsoft/AzureTRE/issues/4006))
+* Fix numbering issue within `bug_report.md` template ([#4028](https://github.com/microsoft/AzureTRE/pull/4028))
```
Comment (tim-allen-ck) [inline CHANGELOG.md:46]: Hi @Sweetdevil144 this needs to be in the 0.19 version section. 
move it to line 14

Comment (Sweetdevil144) [reply CHANGELOG.md]: Okay. My bad :)

**Review by Sweetdevil144 — COMMENTED** [review]
Changes Updated in ChangeLog

File: CHANGELOG.md (hunk 1 above)
Comment (tim-allen-ck) [reply CHANGELOG.md]: did you move it to the right section?

**Review by tim-allen-ck — APPROVED** [review]

**Review by marrobi — APPROVED** [review]

**Pull request merged**